### PR TITLE
Added OTAWebUpdater Doc

### PR DESCRIPTION
**Pull request by karan6190**
Provided the documentation for OTA through Web Browser

### CODE REVIEW

File: libraries/ArduinoOTA/examples/OTAWebUpdater/OTAWebUpdater.ino [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+#include <Update.h>
+
+const char* host = "ESP32";
+const char* ssid = "xxx";
+const char* password = "xxxx";
+
+WebServer server(80);
+
+const char* loginIndex = "<form name='loginForm'>"
+                         "<table width='20%' bgcolor='A09F9F' align='center'>"
```
Comment (me-no-dev) [inline libraries/ArduinoOTA/examples/OTAWebUpdater/OTAWebUpdater.ino]: can you please structure the html code with proper tabs and spaces?

File: docs/OTAWebUpdate/OTAWebUpdate.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+- after deployment if user is unable to expose Firmware for OTA from external update server
+- provide updates after deployment to small quantity of modules when setting an update server is not practicable
+
+## Requirements
+- The ESP and the computer must be connected to the same network
+
+## Implementation
+The sample implementation has been done using:
+- example sketch OTAWebUpdater.ino
+- NodeMCU 1.0 (ESP-12E Module)
```
Comment (copercini) [inline docs/OTAWebUpdate/OTAWebUpdate.md]: ESP-12E ?

**Comment by me-no-dev** [comment]
and there is also [this](https://github.com/espressif/arduino-esp32/blob/master/libraries/WebServer/examples/WebUpdate/WebUpdate.ino)

**Comment by karan6190** [comment]
Updated the format of the code and document

**Comment by me-no-dev** [comment]
@karan6190 here is an example format that I meant :)
```cpp
const char* loginIndex = 
  "<form name='loginForm'>"
    "<table width='20%' bgcolor='A09F9F' align='center'>"
      "<tr>"
        "<td colspan=2><center><font size=4><b>ESP32 Login Page</b></font></center><br></td>"
        "<br>"
        "<br>"
      "</tr>"
      "<tr>"
        "<td>Username:</td>"
        "<td><input type='text' size=25 name='userid'><br></td>"
      "</tr>"
      "<br>"
      "<br>"
      "<tr>"
        "<td>Password:</td>"
        "<td><input type='Password' size=25 name='pwd'><br></td>"
        "<br>"
        "<br>"
      "</tr>"
      "<tr>"
        "<td><input type='submit' onclick='check(this.form)' value='Login'></td>"
      "</tr>"
    "</table>"
  "</form>"
  "<script>"
    "function check(form){"
      "if(form.userid.value=='admin' && form.pwd.value=='admin'){"
        "window.open('/serverIndex')"
      "} else {"
        "alert('Error Password or Username')/*displays error message*/"
      "}"
      "}"
  "</script>";
```

**Comment by me-no-dev** [comment]
Thanks :)

**Pull request merged**